### PR TITLE
bpo-42451: Indicate that PyTuple_GetItem does not support negative indices

### PR DESCRIPTION
**Pull request by 0xYasser**
https://bugs.python.org/issue42451

### CODE REVIEW

File: Doc/c-api/tuple.rst [hunk 1]
```diff
@@ -56,8 +56,10 @@ Tuple Objects
 
 .. c:function:: PyObject* PyTuple_GetItem(PyObject *p, Py_ssize_t pos)
 
-   Return the object at position *pos* in the tuple pointed to by *p*.  If *pos* is
-   out of bounds, return ``NULL`` and set an :exc:`IndexError` exception.
```
Comment (vstinner) [inline Doc/c-api/tuple.rst:60]: You can say:

If pos is negative or out of bound, ...

To document that IndexError is raised if pos is negative.

File: Misc/NEWS.d/next/Documentation/2020-11-26-21-45-09.bpo-42451.i4QoA_.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+update PyTuple_GetItem docs to indicate that it does not support negative
```
Comment (vstinner) [inline Misc/NEWS.d/next/Documentation/2020-11-26-21-45-09.bpo-42451.i4QoA_.rst]: I don't think that it's worth it, you can remove it ;-) I add the skip news label.

**Comment by terryjreedy** [comment]
After I removed the trailing space on line 59, I noticed that it was not added by the PR but existed before.  So a) the previous change to this doc must have been before we checked .rst files and b) there might be other trailing whitespace lines.  If there are, and the whitespace tests fail again, please run `python tools/scripts/patchcheck.py` in the repository directory.  (As described in the devguide somewhere.)

**Comment by terryjreedy** [comment]
Victor, do you otherwise consider this ready to merge?  Want me to do it?

**Review by vstinner — APPROVED** [review]
LGTM.

**Pull request merged**